textdb/textdb-dataflow/pom.xml

```diff
@@ -41,6 +41,11 @@
             <artifactId>textdb-storage</artifactId>
             <version>${textdb.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
```

Comment:
@sandeepreddy602 did we use Mockito before?  If not, should we use it? 

Added @JavierJia